### ISSUE TITLE
Tags pushed by docker manifest not listed 

### ISSUE THREAD

**Issue by BulatSaif** [SUSE/Portus#1813]
Hello,

I try to use [docker manifest](https://docs.docker.com/engine/reference/commandline/manifest/) to push "multi-arch" containers. Docker push the manifest and the Portus get notifications, creates repo and increase numbers of tags, but tags itself is not visible.

As example I try to create "multi-arch" `ubuntu` image
- **Expected behavior**: 
New image appears in Portus   
- **Actual behavior**: 
Portus create new repo, increase tag number
![screenshot from 2018-05-07 18-11-38](https://user-images.githubusercontent.com/24387396/39709361-4ac162c4-5222-11e8-94b5-e0e57049cf8d.png)
but Image tag is not listed
![screenshot from 2018-05-07 18-11-50](https://user-images.githubusercontent.com/24387396/39709302-348be2cc-5222-11e8-970b-c12b68e7434b.png)

### Deployment information

**Deployment method**:
I have VM (ip:10.10.7.30) with latest satbel version of docker
All steps to reproduce bug
``` bsah
$ git clone https://github.com/SUSE/Portus.git
$ cd Portus/examples/compose/secrets
$ openssl req -subj '/CN=Host/O=Name./C=RU' -new -newkey rsa:2048 -days 3650 -nodes -x509 -keyout portus.key -out portus.crt
$ cd ../
$ sed -i 's/MACHINE_FQDN=172.17.0.1/MACHINE_FQDN=10.10.7.30/' .env
$ docker-compose -f docker-compose.insecure.yml  up -d

# Configure UI http://10.10.7.30:3000, 
#  - Create user Admin
#  - Registry 
#    Hostname: 10.10.7.30:5000
#    External Registry Name: localhost:5000

$ docker pull amd64/ubuntu:16.04
$ docker pull arm32v7/ubuntu:16.04
$ docker tag amd64/ubuntu:16.04  localhost:5000/admin/ubuntu-amd64:16.04
$ docker tag arm32v7/ubuntu:16.04  localhost:5000/admin/ubuntu-arm32v7:16.04

$ docker login localhost:5000
$ docker push  localhost:5000/admin/ubuntu-arm32v7:16.04
$ docker push  localhost:5000/admin/ubuntu-amd64:16.04

# need to add `"experimental": "enabled"`  in  `~/.docker/config.json` to run `docker manifest` 
$ docker manifest create --insecure localhost:5000/admin/ubuntu:16.04 \
       localhost:5000/admin/ubuntu-arm32v7:16.04 \
       localhost:5000/admin/ubuntu-amd64:16.04
Created manifest list localhost:5000/admin/ubuntu:16.04

$ docker manifest push localhost:5000/admin/ubuntu:16.04
Pushed ref localhost:5000/admin/ubuntu@sha256:7a3b062531155aa4aee0f0e01a3f4440c86dcf972b1b73ec5d988ae438bb2651 with digest: sha256:7a3b062531155aa4aee0f0e01a3f4440c86dcf972b1b73ec5d988ae438bb2651
Pushed ref localhost:5000/admin/ubuntu@sha256:ff6395f144ccf5dd19e2c1a5df014b183ad9ac4f1c6b22ea5830d2ed05ee03ed with digest: sha256:ff6395f144ccf5dd19e2c1a5df014b183ad9ac4f1c6b22ea5830d2ed05ee03ed
sha256:6fa30188dc6e507bb2e56b3355339340dc243ff01f4022ffa160de9daf340a45

```

**Docker version**:  
``` bash
$ docker version 
Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   9ee9f40
 Built:        Thu Apr 26 07:17:20 2018
 OS/Arch:      linux/amd64
 Experimental: true
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   9ee9f40
  Built:        Thu Apr 26 07:15:30 2018
  OS/Arch:      linux/amd64
  Experimental: false
```
**Portus version**:  
2.4.0-dev@9ee98db159fa469951d009bcaecee19a6643bade 
I try it with 2.3 as well

**Comment by str1k3r** [comment]
@BulatSaif have you tried with 2.3.x? any luck? having the same problem.

**Comment by BulatSaif** [comment]
@str1k3r  Yes, I tried with 2.3.x, had same problem. 
Also I tried to sync database and registry, but while doing that I killed database. ( I am new on Portus, so I didn't find command to force the [background process](http://port.us.org/docs/background.html) to start sync. And I thought if I delete database files and restart docker-compose it should run sync task as described [here](http://port.us.org/docs/Configuring-Portus.html#background-process), but I lost  all Namespaces and repositorys, only global  Namespaces was sync)
